Weights/include/CGAL/Weights.h

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2020 GeometryFactory SARL (France).
```

Comment:
Is this a convenience header?  Does it then not need a doxygen comment.  If not a convenience header who needs this?

Reply:
Why does it need any doxygen? It is a convenience header that is there to let users include all weights altogether if necessary. As far as I know, it is a common practice for many CGAL packages. Or maybe I did not fully understand your comment?

Reply:
It should be similar to [`CGAL/meshsegmentation.h`](https://github.com/CGAL/cgal/blob/master/Surface_mesh_segmentation/include/CGAL/mesh_segmentation.h#L19)

Reply:
Ok, I have added a Doxygen comment.

Reply:
But I can't find it in the navigation bar of the documentation.

Reply:
But [mesh segmentation](https://doc.cgal.org/latest/Surface_mesh_segmentation/group__PkgSurfaceMeshSegmentationRef.html) also does not show its `mesh_segmentation.h`.

Reply:
That is what I just wrote.   When you follow the [link ](https://github.com/CGAL/cgal/blob/master/Surface_mesh_segmentation/include/CGAL/mesh_segmentation.h#L19) there is a `@file` but with no effect . This is probably a general setting of the CGAL project.

Reply:
Fixed now.